backend/db/ddl-19-concept_graph.jinja.sql

```diff
@@ -1,6 +1,6 @@
-DROP TABLE IF EXISTS concept_graph CASCADE;
+DROP TABLE IF EXISTS {{schema}}concept_graph{{optional_suffix}} CASCADE;
```

Comment:
## Bugfix: `concept_graph` DDL
This was created in November but I'm noticing it now as I'm continuing to make sure that the counts refresh test passes. I'm surprised this didn't come up when I manually updated counts last month.

I added:
- `{{schema}}`
- `{{optional_suffix}}`
- `{{optional_index_suffix}}`